sdcm/utils.py

```diff
@@ -888,3 +888,7 @@ def inner(*args, **kwargs):
                     cls_self.version,
                     cls_self.__class__.__name__))
         return inner
+
+
+class PrometheusSnapshotErrorException(Exception):
```

Comment:
we need a better place to store all the exceptions which SCT throws... currently better to put this in the nearest place in the code that throws it

Reply:
Thought about separate file.

Reply:
fixed